src/Mpi/MpiControl.cpp

```diff
@@ -408,7 +408,7 @@ template <int ndim>
 void MpiControl<ndim>::UpdateSinksAfterAccretion
  (Sinks<ndim>* sink)                             ///< [inout] Pointer to sinks array
 {
-  const int number_variables = ndim*8 + 11;      // ..
+  const int number_variables = ndim*6 + 15;      // ..
```

Comment:
Is this ndim*8 + 11 --> ndim*6 +15 due to the merge? It should be just the addition of one varaible

Reply:
This takes into account your fix (which I hadn't updated to), plus the dmdt variable.

Reply:
Ok